resources/l10n/da/LC_MESSAGES/loot.po

```diff
@@ -4,1267 +4,2086 @@
-# File: src/gui/html/js/translateStaticText.js, line: 485
-msgid "No plugin selected"
-msgstr "Intet plugin valgt"
+#: src/gui/qt/settings/new_game_dialog.cpp:114
+msgid "A game with this folder already exists."
+msgstr "Et spil med denne mappe eksistere allerede."
 
 #
-# File: src/gui/html/js/translateStaticText.js, line: 455
-# File: src/gui/html/js/translateStaticText.js, line: 455
-msgid "No text specified"
-msgstr "Ingen tekst angivet"
+# File: src/gui/html/js/translateStaticText.js, line: 413
+# File: src/gui/html/js/translateStaticText.js, line: 533
+# File: src/gui/html/js/translateStaticText.js, line: 413
+# File: src/gui/html/js/translateStaticText.js, line: 533
+#: src/gui/qt/settings/settings_dialog.cpp:110
+msgid "Settings"
+msgstr "Indstillinger"
 
-#
-# File: src/gui/html/js/translateStaticText.js, line: 248
-# File: src/gui/html/js/translateStaticText.js, line: 248
-msgid "Note"
-msgstr "Notits"
+#: src/gui/qt/settings/settings_dialog.cpp:112
+msgid "General"
+msgstr "Generel"
 
-#
-# File: src/gui/html/js/translateStaticText.js, line: 656
-# File: src/gui/html/js/translateStaticText.js, line: 656
-msgid "OK"
-msgstr "OK"
+#: src/gui/qt/settings/settings_dialog.cpp:114
+msgid "Add new game..."
+msgstr "Tilføj nyt spil..."
 
 #
 # File: src/gui/cef/query/query.h, line: 84
 # File: src/gui/cef/query/query.h, line: 84
-#, fuzzy
+#: src/gui/query/types/apply_sort_query.h:96
 msgid ""
 "Oh no, something went wrong! This is usually because \"%1%\" is set to be "
 "read-only. If it is, unset it and try again. If it isn't, you can check your "
 "LOOTDebugLog.txt (you can get to it through the main menu) for more "
 "information."
 msgstr ""
-"Åh nej, noget gik galt! [[This is usually because \"%1%\" is set to be read-"
-"only. If it is, unset it and try again.]] Hvis det ikke er derfor kan du "
-"kigge i din LOOTDebugLog.txt (du kan finde den via hovedmenuen) for mere "
-"information."
+"Åh nej, noget gik galt! Dette er normalt fordi \"%1%\" er indstillet til at "
+"være skrivebeskyttet. Hvis dette er tilfældet, skal du ændre indstillingen "
+"og prøve igen. Hvis det ikke er tilfældet kan du kigge i din LOOTDebugLog."
+"txt (du kan finde den via hovedmenuen) for mere information."
 
 #
-# File: src/gui/cef/query/query.h, line: 50
-# File: src/gui/cef/query/query.h, line: 50
-msgid ""
-"Oh no, something went wrong! You can check your LOOTDebugLog.txt (you can "
-"get to it through the main menu) for more information."
-msgstr ""
-"Åh nej, noget gik galt! Du kan kigge i din LOOTDebugLog.txt (du kan finde "
-"den via hovedmenuen) for mere information."
-
 #
-# File: src/gui/html/js/translateStaticText.js, line: 392
-# File: src/gui/html/js/translateStaticText.js, line: 392
-msgid "Open Debug Log Location"
-msgstr "Åben sted for fejlsøgningslog"
+#: src/gui/query/types/get_game_data_query.h:47
+msgid "Parsing, merging and evaluating metadata..."
+msgstr "Læser, sammenfletter og evaluerer metadata…"
```

Comment:
You type ellipses as three separate dots in most other strings, so this should probably be the same for consistency.

Reply:
Resolved on discord